app/main/index.ts

```diff
@@ -279,7 +285,7 @@ app.on('ready', () => {
 		AppMenu.setMenu(props);
 		const activeTab = props.tabs[props.activeTabIndex];
 		if (activeTab) {
-			mainWindow.setTitle(`Zulip - ${activeTab.webview.props.name}`);
+			mainWindow.setTitle(`Zulip - ${activeTab.props.name}`);
```

Comment:
It seems on the BrowserView, it doesn't auto decode encoded text (in case of abhigyank.zulipdev.org) so we see escaped HTML on the title. For temporary fix we can decode the title here with a comment and create an issue upstream to decode the url. It used to work webview so should work here as well.

Reply:
Fixed this in the latest commit. We weren't decoding it when we were pushing it to tabs.